Resources/Prototypes/Nyanotrasen/Roles/Jobs/Epistemics/forensicmantis.yml

```diff
@@ -3,7 +3,6 @@
   name: job-name-mantis
   description: job-description-mantis
   playTimeTracker: JobForensicMantis
-  setPreference: false # Parkstation-Playtime
```

Comment:
Let's just set this to true instead of removing the line. No need to obfuscate it.

Reply:
I added the line unintentionally before